dataminer-overview/DataMiner_Solutions/DOM Viewer/domviewer_about.md

```diff
@@ -0,0 +1,15 @@
+---
+uid: domviewer_about
+---
+
+# DOM Viewer
+
+The DOM Viewer is a cutting-edge low-code application designed to provide unparalleled insight in and control over your DataMiner Object Model (DOM) definitions and instances.
+
+Imagine being able to visualize complex DOM relationships effortlessly. With the DOM Viewer, you can explore how DOM definitions interconnect through an intuitive node-edge graph. You can also instantly access detailed information by hovering over definitions, including the module they belong to, the number of sections and fields associated with them, and more.
+
+You can also dive deeper by opening definition tables, where you can see a list of field descriptors filtered by section. You will be able to gain comprehensive insights into field details such as type, description, whether the field is optional or mandatory, and its read-only status. With the state transition graph feature, you can also easily view the life cycle of instances and check how they move between states.
+
+For a broad overview, the DOM Viewer’s powerful table functionality lists all field descriptors across all definitions in every module, complete with extensive filtering options. The same robust features apply to all DOM instances, providing you with a holistic view of your entire DataMiner System.
+
+Deploy the DOM Viewer today to transform your data management experience and unlock the full potential of your DataMiner System.
```

Comment:
Can this app already be deployed? If yes, we should add a link to the catalog page here, and possibly also a tip linking to [Deploying a Catalog item to your system](https://docs.dataminer.services/user-guide/Cloud_Platform/Catalog/Deploying_A_DataMiner_Connector_to_your_system.html). If no, it would be better to rephrase this so that it's clear this is not available yet.